<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mode_imu</class>
 <widget class="QWidget" name="Mode_imu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Raw</string>
   </property>
   <widget class="QLineEdit" name="tb_my">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_mx">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_mz">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mxyz:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_ay">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_ax">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_az">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Axyz:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_gx">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_gz">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gxyz:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_gy">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_tilt_raw">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tilt:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_change_mode_imu">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Mode IMU</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>591</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Parameters:</string>
   </property>
   <widget class="QLineEdit" name="tb_gbelieve">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_my_scale">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>G believe:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_46">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mxyz scale:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mxyz bias:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_mz_scale">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_gx_bias">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_my_bias">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_mz_bias">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_angle_adjust">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_mx_bias">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gxyz bias:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_gy_bias">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_gz_bias">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_mx_scale">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tilt offset:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_tilt">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_52">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tilt:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_ay_bias">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Axyz bias:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_ax_bias">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_az_bias">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_complementary_gain">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Madgwick beta:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_madgwick_beta">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Complementary gain:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_mode_imu_load_params">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mode_imu_write_params">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_mode_imu_save_params">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Calibrated</string>
   </property>
   <widget class="QLineEdit" name="tb_cab_mz">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_cab_my">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_cab_mx">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_cab_az">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_cab_ay">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_cab_ax">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Axyz:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mxyz:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_cab_gz">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_cab_gx">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Gxyz:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_cab_gy">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_tilt_cal">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tilt:</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_tilt_calib">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>T Calibrate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gyro_calib_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>M Calibrate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gyro_calib_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>A Calibrate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gyro_calib">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>G Calibrate</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>RPY</string>
   </property>
   <widget class="QLineEdit" name="tb_yaw">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_pitch">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="tb_roll">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RPY:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
